config.sample.yml

```diff
@@ -13,17 +13,10 @@ temporal:
     maximumAttempts: 5
     backoffCoefficient: 2.0
 
-# Cache configuration
-cache:
-  enabled: true
-  databasePath: "./workflow_cache.db"
```

Comment:
Since we are getting rid of the DB, could we remove the sqlite dependency too? Thanks! 😃